content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -176,27 +176,27 @@ const sdk = new opentelemetry.NodeSDK({
 });
 
 sdk.start()
-{{< /ot-tab >}}
+{{< /tab >}}
 
-{{< /ot-tabs >}}
+{{< /tabpane >}}
 
 ### Run Application
 
 Now you can run your application as you normally would, but you can use the `--require` flag to load the tracing code before the application code.
 
-{{< ot-tabs TypeScript JavaScript >}}
+{{< tabpane >}}
```

Comment:
Make changes to this tabpane element similar to the above one that has lang=console.